public/assets/js/main.js

```diff
@@ -20,18 +22,53 @@
         }
     })();
 
-    (function selectInputContent() {
+    (function selectTextCopyToClipboard() {
         var selector = 'input[type="text"]';
         var el = document.querySelectorAll(selector);
+        var origHelpBlockText = 'Click to copy';
```

Comment:
Were we to abstract the pug blocks to an include (as suggested below), it might be better to set this by reading what's currently set in the element, so that we only have to change it on place, should we decide on different messaging in the future -- not that I foresee doing that, but still.